joplin/settings.py

```diff
@@ -305,7 +305,7 @@ def show_toolbar(request):
 BASE_URL = 'https://austintexas.io'
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = os.getenv('DJANGO_SECRET_KEY')
+SECRET_KEY = os.getenv('DJANGO_SECRET_KEY', 'fake_key')
```

Comment:
hooray! I've been doing silly things all over the place to get this working, didn't think to go to the getenv part